README.md

```diff
@@ -1,7 +1,7 @@
 # Delete multiple branches GitHub Action
 
 An action that deletes multiple branches from repository.
-Optionally one can provide a `prefix` or `suffix` strings that would be appended or prepended to every branch name.
```

Comment:
Why removing prefix here?

Reply:
The previous prefix was replaced with lookups against all the repo branches. The previous implementation users of the action had to explicitly know the branch name.